Extensions/Multiplayer/multiplayertools.ts

```diff
@@ -1519,6 +1597,166 @@ namespace gdjs {
       );
     };
 
+    const onLobbyQuickJoinFinished = (runtimeScene: gdjs.RuntimeScene) => {
+      _isQuickJoiningOrStartingAGame = false;
+      _actionAfterJoiningLobby = null;
+      gdjs.multiplayerComponents.displayLoader(runtimeScene, false);
+    };
+
+    const doQuickJoinLobby = async (
+      runtimeScene: gdjs.RuntimeScene,
+      displayLoader: boolean,
+      openLobbiesPageIfNotEnoughPlayers: boolean
+    ) => {
+      if (_isQuickJoiningOrStartingAGame) return;
+      const _gameId = gdjs.projectData.properties.projectUuid;
+      if (!_gameId) {
+        handleLobbiesError(
+          runtimeScene,
+          'The game ID is missing, the quick join lobby action cannot continue.'
+        );
+        return;
+      }
+
+      _quickJoinLobbyFailureReason = null;
+      _isQuickJoiningOrStartingAGame = true;
+      if (displayLoader) {
+        gdjs.multiplayerComponents.displayLoader(runtimeScene, true);
+      }
+
+      const quickJoinLobbyRelativeUrl = `/play/game/${_gameId}/public-lobby/action/quick-join`;
+      const platformInfo = runtimeScene.getGame().getPlatformInfo();
+
+      try {
+        const quickJoinLobbyResponse: QuickJoinLobbyResponse = await gdjs.evtTools.network.retryIfFailed(
+          { times: 2 },
+          () =>
+            fetchAsPlayer({
+              relativeUrl: quickJoinLobbyRelativeUrl,
+              method: 'POST',
+              dev: isUsingGDevelopDevelopmentEnvironment,
+              body: JSON.stringify({
+                isPreview: runtimeScene.getGame().isPreview(),
+                gameVersion: runtimeScene.getGame().getGameData().properties
+                  .version,
+                supportedCompressionMethods:
+                  platformInfo.supportedCompressionMethods,
+              }),
+            })
+        );
+
+        if (quickJoinLobbyResponse.status === 'full') {
+          gdjs.multiplayerComponents.showNotification({
```

Comment:
did we eventually say we would not show that notification? And instead open the lobbies window in that case if the action param is true? (make the param `openLobbiesPageIfNotEnoughPlayers` a more generic `openLobbiesWindowIfCannotQuickJoin`)

Also, if we decide to show that notification anyway, probably more of a type: warning instead of error? (unsure)

Reply:
If it's full, there's little interest in opening the lobbies and see that they're full, I doubt that players will wait for a lobby to free up